packages/spatial/src/renderer/effects/PostProcessing.ts

```diff
@@ -401,52 +443,52 @@ export type EffectPropsSchema = {
   [Effects.SSGIEffect]: SSGIEffectProps
   [Effects.TRAAEffect]: TRAAEffectProps
   [Effects.MotionBlurEffect]: MotionBlurEffectProps
-  // [Effects.ChromaticAberrationEffect]: ChromaticAberrationEffectProps
-  // [Effects.ColorAverageEffect]: ColorAverageEffectProps
-  // [Effects.DotScreenEffect]: DotScreenEffectProps
-  // [Effects.TiltShiftEffect]: TiltShiftEffectProps
-  // [Effects.GlitchEffect]: GlitchEffectProps
-  // [Effects.GodRaysEffect]: GodRaysEffectProps
-  // [Effects.GridEffect]: GridEffectProps
-  // [Effects.LUT1DEffect]: LUT1DEffectProps
-  // [Effects.LUT3DEffect]: LUT3DEffectProps
-  // [Effects.NoiseEffect]: NoiseEffectProps
-  // [Effects.PixelationEffect]: PixelationEffectProps
-  // [Effects.ScanlineEffect]: ScanlineEffectProps
-  // [Effects.ShockWaveEffect]: ShockWaveEffectProps
-  // [Effects.FXAAEffect]: FXAAEffectProps
-  // [Effects.TextureEffect]: TextureEffectProps
+  [Effects.ChromaticAberrationEffect]: ChromaticAberrationEffectProps
+  [Effects.ColorAverageEffect]: ColorAverageEffectProps
+  [Effects.DotScreenEffect]: DotScreenEffectProps
+  [Effects.TiltShiftEffect]: TiltShiftEffectProps
+  [Effects.GlitchEffect]: GlitchEffectProps
+  //[Effects.GodRaysEffect]: GodRaysEffectProps
+  [Effects.GridEffect]: GridEffectProps
+  [Effects.LUT1DEffect]: LUT1DEffectProps
+  [Effects.LUT3DEffect]: LUT3DEffectProps
+  [Effects.NoiseEffect]: NoiseEffectProps
+  [Effects.PixelationEffect]: PixelationEffectProps
+  [Effects.ScanlineEffect]: ScanlineEffectProps
+  [Effects.ShockWaveEffect]: ShockWaveEffectProps
+  [Effects.FXAAEffect]: FXAAEffectProps
+  [Effects.TextureEffect]: TextureEffectProps
   [Effects.VignetteEffect]: VignetteEffectProps
-  // [Effects.LensDistortionEffect]: LensDistortionEffectProps
+  [Effects.LensDistortionEffect]: LensDistortionEffectProps
 }
 
 export type EffectPropsSchemaType = (typeof defaultPostProcessingSchema)[keyof typeof defaultPostProcessingSchema]
 
 export const defaultPostProcessingSchema: EffectPropsSchema = {
```

Comment:
Were you able to check if the default values for the other effects matched the default values from the library/demos?